projects/1.18/assets/botania/botania/lang/zh_cn.json

```
@@ -1870,212 +2025,269 @@
   "botania.page.elvenLenses10": "闪耀透镜",
   "botania.page.elvenLenses11": "$(item)绊线透镜$(0)是另一种$(thing)控制透镜$(0)。$(p)装有此透镜的发射器只会在有生物或玩家经过其$(thing)魔力脉冲$(0)的轨迹时才会发射。",
   "botania.page.elvenLenses12": "绊线透镜",
+
   "botania.entry.prism": "魔力棱镜",
   "botania.tagline.prism": "飞行中改变脉冲性质",
   "botania.page.prism0": "$(item)魔力透镜$(0)的原理使得能量可以通过控制$(thing)魔力脉冲$(0)传递，$(item)魔力棱镜$(0)使用该技术使得脉冲在穿过时附加其上的透镜效果。$(p)使用$(item)幻影平台$(0)作为原料，这个方块不具有碰撞体积，实体和脉冲可以穿过它，就像它不存在一般",
   "botania.page.prism1": "使用任何类型的$(item)魔力透镜$(0)右击它可以将其装在棱镜上。当$(thing)魔力脉冲$(0)穿过棱镜时，它的颜色和效果就会被设置成透镜的。假如透镜未染色，脉冲就会变为白色。$(p)使用4个基础透镜同样可以增加脉冲损失$(thing)魔力$(0)前飞行的时间，让它飞得更远。",
   "botania.page.prism2": "它也可以用红石信号关闭。",
   "botania.page.prism3": "墙上另一个像素",
+
   "botania.entry.poolCart": "轨道上的魔力",
   "botania.tagline.poolCart": "这不是网络框架",
   "botania.page.poolCart0": "说起远距离传输$(thing)魔力$(0)，还有什么比把$(item)魔力池$(0)装在$(item)矿车$(0)里更简单的方法呢？这很简单。就像其他的矿车一样，制造$(item)魔力$(0)$(item)池矿车$(0)，它可以在轨道上装载并运输$(thing)魔力$(0)。$(p)破坏装有$(thing)魔力池$(0)的矿车会浪费掉$(thing)魔力$(0)，小心点。",
   "botania.page.poolCart1": "在矿车和池子间传输$(thing)魔力$(0)需要$(item)魔力泵$(0)。它可以面对4个主要方向并在相邻的池子和目标池子对面轨道上的魔力池矿车之间传输$(thing)魔力$(0)。$(p)传输速率由于本身构造相当的快，所以作为高速的代价，它伴随着$(thing)魔力损耗$(0)。",
   "botania.page.poolCart2": "在泵中，$(thing)魔力$(0)流动去向的一侧是粉红色的，$(thing)魔力$(0)会从蓝色端吸取。$(p)泵可以用红石关闭，$(item)比较器$(0)会根据矿车内$(thing)魔力$(0)含量输出信号。$(p)最后，魔力池矿车只能用于运输，不能将其中的$(thing)魔力$(0)接收，发送，传输或注入任何物品。",
   "botania.page.poolCart3": "我爱火车",
   "botania.page.poolCart4": "这不是管道",
+
   "botania.entry.sparkChanger": "火花工匠台",
   "botania.tagline.sparkChanger": "用红石加强对火花的控制",
   "botania.page.sparkChanger0": "$(item)火花工匠台$(0)是个能改变$(item)火花升级$(0)的方块，只要将其放在$(item)魔力池$(0)或其他接有$(item)火花$(0)的方块上就能让它们相连接。$(p)只要用$(item)火花升级$(0)右击$(item)火花工匠台$(0)就能将升级装入，再次右击即可取下。",
   "botania.page.sparkChanger1": "当$(item)火花工匠台$(0)收到红石信号时，它会在相连的$(item)火花$(0)中随机挑选一个与自身所存升级类型不同的火花，并用存储的升级替换$(item)该火花$(0)的升级。$(p)储存的升级可以通过$(item)漏斗$(0)一类的途径导出，使用比较器也可以辨别升级的类型。",
   "botania.page.sparkChanger2": "修改你的火花",
+
   "botania.entry.bellows": "魔源风箱",
   "botania.tagline.bellows": "让魔力池和熔炉更快工作",
   "botania.page.bellows0": "$(item)魔源风箱$(0)是一种简单的，用来提升$(item)魔力池$(0)传输速率的设备。$(p)将这些风箱放在魔力池旁边，并将风口对准，$(item)魔力池$(0)传输$(thing)魔力$(0)到$(item)魔力石板$(0)或类似物品的速率就会提升。使用多个风箱将使得传输速率变得更快",
   "botania.page.bellows1": "$(item)魔源风箱$(0)另一个关注度不高的能力是让$(item)熔炉$(0)的火烧的更旺$(p)通过右击来手动使用$(item)魔源风箱$(0)，并将风向口对准一个正在使用$(item)熔炉$(0)并鼓动他们能够使得$(item)熔炉$(0)烧东西变得更有效率且更为迅速。",

```

```suggestion
  "botania.page.bellows1": "$(item)魔源风箱$(0)另一个关注度不高的能力是让$(item)熔炉$(0)的火烧的更旺。$(p)通过右击来手动使用$(item)魔源风箱$(0)，将风向口对准一个工作中的$(item)熔炉$(0)，并鼓动风箱即可使$(item)熔炉$(0)的冶炼速率更高。
```
大致改了一下这个句子，可能改的并不是很好